src/current/_includes/releases/v23.2/v23.2.18.md

```diff
@@ -0,0 +1,14 @@
+## v23.2.18
+
+Release Date: December 26, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-2-18-sql-language-changes">SQL language changes</h3>
+
+- Added the `legacy_varchar_typing` session setting. When set to `on`, type-checking and overload resolution for comparisons involving `VARCHAR` columns ignore the newly added `unpreferred` overload preference. The `unpreferred` preference introduced a change in `VARCHAR` typing behavior. For more information, see [#133037][#133037] and [#137837][#137837]. The `legacy_varchar_typing` session setting is `on` by default. [#137945][#137945]
```

Comment:
I don't think this should mention "unpreferred" or "overloads"—those are implementation details. Those shouldn't have been in the release notes, apologies for that. How about something like this instead:

> Added the `legacy_varchar_typing` session setting. When set to `on`, type-checking comparisons involving `VARCHAR` columns behave as they did in all previous versions. When set to `off`, type-checking of these comparisons is more strict and queries that previously succeeded may now error with the message `unsupported comparison operator`. These errors can be fixed by added explicit type casts. The `legacy_varchar_typing` session setting is `on` by default.